src/components/Header/Header.jsx

```diff
@@ -1,6 +1,12 @@
-// import the React library here
-// import a css file containig article styles
+import React from 'react';
+import './Header.css';
+import Navigation from '../Navigation/Navigation';
 
-// Create a Header function returning the HTML of header block
+const Header = () => (
```

Comment:
you can use export here
```suggestion
export const Header = () => (
```

Reply:
I've changed everything like you asked, but when I am trying to deploy, it says that I should use export default instead